packages/ibm-products/src/global/js/package-settings.js

```diff
@@ -73,6 +73,8 @@ const defaults = {
     ButtonMenu: false,
 
     // Novice to pro components not yet reviewed and released:
+    Carousel: false,
```

Comment:
If the `Carousel` is meant to be internal, I don’t think we need to add it to `package-settings.js` at all? I think there’s a few other instances you can follow from our internal components. 🙂